### PR TITLE
update to v3.21.0 of the terraform provider

### PR DESCRIPTION
**Pull request by jaxxstorm**
This change also disables the tests dedicated hosts until #1264 is figured out

### CODE REVIEW

**Comment by stack72** [comment]
> This change also disables the dedicated hosts until #1264 is figured out

I believe you mean the tests for the dedicated hosts - not the actual resources themselves right?

**Comment by jaxxstorm** [comment]
Yep, fixed, sorry for the confusion

**Review by komalali — APPROVED** [review]

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -24,6 +25,7 @@ func TestAccDedicatedHosts(t *testing.T) {
 
 	integration.ProgramTest(t, &test)
 }
+*/
```
Comment (komalali) [inline examples/examples_nodejs_test.go:28]: Is this meant to be excluded? If yes, could we add a note for why?

Comment (jaxxstorm) [reply examples/examples_nodejs_test.go:28]: done!

**Pull request merged**